### ISSUE TITLE
namedParameters with typedoc 0.20

### ISSUE THREAD

**Issue by ozum** [tgreyuk/typedoc-plugin-markdown#183]
Hi,

TypeDoc seems to solve `namedParameters` problem. They dropped `excludeNotExported` parameter.

Now, default TypeDoc renders named parameters correctly. However, the same code cannot be rendered using the markdown plugin. Please see the code below and screenshots of outputs:

```ts
/**
 * FOO
 * @param __namedParameters various options
 * @param foo An interesting value
 * @param bar Another value
 */
export function demo({ foo = 42, bar = 43 }: { foo?: number; bar?: number } = {}) {}

/**
 * FOO
 * @param __namedParameters various options
 */
export function demo2({
  foo = 42,
  bar = 43,
}: {
  /** hhh */
  foo?: number;
  /** hhhdjf */
  bar?: number;
} = {}) {}
```

# TypeDoc
![image](https://user-images.githubusercontent.com/1497060/105197449-aee84880-5b4d-11eb-8829-036da6a81d80.png)

# Markdown Plugin
![image](https://user-images.githubusercontent.com/1497060/105197493-b90a4700-5b4d-11eb-8fd6-83d9c4b7c802.png)

**Comment by tgreyuk** [comment]
thanks @ozum . We would have to extract `__namedParameters` from the table. Do you think this layout works?

![image](https://user-images.githubusercontent.com/11680870/105252245-0057fe00-5b75-11eb-8287-fd1d89b0c348.png)

**Comment by tgreyuk** [comment]
@ozum available in typedoc-plugin-markdown@3.4.2 .

**Comment by ozum** [comment]
Many thanks.
